docs/source/top_level_elements/relatedItem.rst

```diff
@@ -0,0 +1,702 @@
+
+This XPath appears once, in the record for the Charles Dabny collection; i.e. `collections:dabney <https://digital.lib.utk.edu/collections/islandora/object/collections:dabney/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Collection" type="host">
+      <titleInfo>
+        <title>University of Tennessee President's Papers, 1867-1954</title>
+      </titleInfo>
+      <identifier>AR.0001</identifier>
+      <location>
+        <physicalLocation authority="naf" valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+      </location>
+    </relatedItem>
+
+relatedItem/location
+--------------------
+
+This XPath `relatedItem/location/url` is used 8516 times, but only uses 33 distinct strings; e.g. `ruskin:204 <https://digital.lib.utk.edu/collections/islandora/object/ruskin:204/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Collection" type="host">
+      <titleInfo>
+        <title>Ruskin Cooperative Association Collection</title>
+      </titleInfo>
+      <identifier>MS.0023</identifier>
+      <location>
+        <url>https://n2t.net/ark:/87290/v81g0jf1</url>
+      </location>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> dbo:isPartOf <https://n2t.net/ark:/87290/v81g0jf1> ;
+        dbo:collection "Ruskin Cooperative Association Collection, MS.0023" .
+
+relatedItem/abstract
+--------------------
+
+`relatedItem/abstract` is used 865 times, across several collections; e.g. `sanborn:1196 <https://digital.lib.utk.edu/collections/islandora/object/sanborn:1196/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+        <title>Sanborn Fire Insurance Maps Collection</title>
+      </titleInfo>
+      <location>
+        <url>http://digital.lib.utk.edu/collections/sanbornmapcollection</url>
+      </location>
+      <abstract>The Sanborn Fire Insurance Maps are a large-scale representation of the growth and layout of American cities. The University of Tennessee Libraries' collection currently provides digital versions of the four earliest sets of Knoxville maps - 1884, 1890, 1903, and 1917.</abstract>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix rdfs: <https://www.w3.org/TR/rdf-schema/> .
+    @prefix pcdm: <http://pcdm.org/models#> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/collection/1> a pcdm:Collection ;
+      rdfs:label "Sanborn Fire Insurance Maps Collection" ;
+      dcterms:abstract "The Sanborn Fire Insurance Maps are a large-scale representation of the growth and layout of American cities. The University of Tennessee Libraries' collection currently provides digital versions of the four earliest sets of Knoxville maps - 1884, 1890, 1903, and 1917." .
```

Comment:
Again, we'd migrate metadata attached to a collection object following this same doc.  I'd drop this unless it conflicts.

Reply:
@markpbaggett I hear what you're saying.

Reply:
I have always dropped this when remediating collections, so I'd say that dropping will also help us with consistency.